### PR TITLE
fix(presto-client): improve published package information

### LINKED ISSUE #4: License, repository and tags are missing
### Current Behavior

`@prestodb/presto-js-client` NPM package does not have the next information:
- License
- Repository
- Tags

### Expected Behavior

`@prestodb/presto-js-client` NPM package should have the next information:
- License
- Repository
- Tags

### GitHub Repo

_No response_

### Steps to Reproduce

1. Go to [the next url](https://www.npmjs.com/package/@prestodb/presto-js-client/v/0.1.0).
2. Notice how the License, Repository and Tags are missing:

image

### Additional Information

_No response_

### PR DESCRIPTION
**Pull request by alonsovb**
## Changes

- Fixes #4 by moving some README.md and CONTRIBUTING.md files around.
- Solves #6 by deprecating the interval property and instead just by default delaying retries when server is overloaded (returning a 503 status response).
- Remove source maps from the build so it's not included in the published package (it causes a warning when using the published package).

## Notes

None

## Screenshots

None

### CODE REVIEW

**Comment by alonsovb** [comment]
I mixed up 2 different fixes (#4 and #6) plus a bit of extra improvements on the same PR, sorry about that
If required, I can split into 2 branches

File: presto-client/src/client.ts [hunk 1]
```diff
@@ -91,11 +94,25 @@ export class PrestoClient {
...
         throw new Error(`Query failed: ${JSON.stringify(await response.text())}`)
       }
 
       const prestoResponse = (await response.json()) as PrestoResponse
+      if (!prestoResponse) {
+        throw new Error(`Query failed with an empty response from the server.`)
+      }
+
+      const { error } = prestoResponse
+      if (error) {
```
Comment (aarong506) [inline presto-client/src/client.ts]: if error is only used here maybe it is better to 
```suggestion
      if (prestoResponse.error) {
```

That way we remove the variable from the scope

File: presto-client/src/client.ts [hunk 2]
```diff
@@ -91,11 +94,25 @@ export class PrestoClient {
...
       if (response.status !== 200) {
         throw new Error(`Query failed: ${JSON.stringify(await response.text())}`)
       }
 
       const prestoResponse = (await response.json()) as PrestoResponse
+      if (!prestoResponse) {
+        throw new Error(`Query failed with an empty response from the server.`)
+      }
+
+      const { error } = prestoResponse
```
Comment (aarong506) [inline presto-client/src/client.ts]:
```suggestion
```

**Comment by aarong506** [comment]
@tdcmeehan 
Perhaps you could enable at least one review request before allowing a merge.

File: apps/nest-server/src/app/app.service.ts [hunk 3]
```diff
@@ -5,14 +5,22 @@ import PrestoClient, { PrestoClientConfig } from '@prestodb/presto-js-client'
 export class AppService {
   async getData(): Promise<unknown> {
     const clientParams: PrestoClientConfig = {
-      catalog: 'tpcds',
+      catalog: 'tpch',
       host: 'http://localhost',
       port: 8080,
       schema: 'sf10',
```
Comment (yhwang) [inline apps/nest-server/src/app/app.service.ts]: for the `tpch` catalog, the `sf10` schema doesn't exist:
```
presto:tpch> show schemas;
       Schema
--------------------
 information_schema
 sf1
 sf100
 sf1000
 sf10000
 sf100000
 sf300
 sf3000
 sf30000
 tiny
(10 rows)
```

File: presto-client/README.md [hunk 4]
```diff
@@ -1,106 +1,104 @@
...
 
-   ```bash
-   npm run publish presto-client -- --releaseAs=premajor|preminor|prepatch --preid=beta
-   ```
+```typescript
+const client = new PrestoClient({
+  catalog: 'tpcds',
+  host: 'http://localhost',
+  port: 8080,
+  schema: 'sf10',
```
Comment (yhwang) [inline presto-client/README.md]: here is the list of built-in schemas that `tpcds` catalog provides:
```
presto:tpcds> show schemas;
       Schema
--------------------
 information_schema
 sf1
 sf100
 sf1000
 sf10000
 sf100000
 sf300
 sf3000
 sf30000
 tiny
```

File: presto-client/package.json [hunk 5]
```diff
@@ -6,5 +6,20 @@
   },
   "type": "commonjs",
   "main": "./src/index.js",
-  "typings": "./src/index.d.ts"
+  "typings": "./src/index.d.ts",
```
Comment (jorgeramirezamora) [inline presto-client/package.json]: Maybe worth review the order of these keys based on the order in https://docs.npmjs.com/cli/v10/configuring-npm/package-json 

You might use https://www.npmjs.com/package/sort-package-json

**Review by jorgeramirezamora — COMMENTED** [review]
Just a couple of comments. For now is good to do a single PR with all these changes, but for semantic release is always better to do smaller PRs so SHE can better infer the next version

File: package.json [hunk 6]
```diff
@@ -1,14 +1,18 @@
 {
   "name": "presto-js-client",
+  "private": true,
```
Comment (jorgeramirezamora) [inline package.json]: This should be public

File: CONTRIBUTING.md [hunk 7]
```diff
@@ -0,0 +1,138 @@
+# presto-client
```
Comment (jorgeramirezamora) [inline CONTRIBUTING.md]: Maybe this should like under `presto-client/CONTRIBUTING.md`

**Review by jorgeramirezamora — COMMENTED** [review]
A couple of comments

**Review by jorgeramirezamora — APPROVED** [review]
LGTM!

**Review by yhwang — APPROVED** [review]
LGTM

**Pull request merged**